src/routes/play/+page.server.ts

```diff
@@ -35,7 +35,7 @@ export async function load({ url, cookies }) {
 
     const random = daily_mode ?
         () => prand.unsafeUniformIntDistribution(0, possible_locations.length - 1, rng) :
```

Comment:
Looks like the docs for the prand library say that both limits are inclusive, so I don't think the OOBE exists here